docs/concepts/event-driven-architecture.md

```diff
@@ -33,10 +33,11 @@ Before getting started, let's define some important concepts and terms related t
 
 To implement event-driven architecture in your Amplication project, you'll need to:
 
-1. Create a message broker
+1. Create a message broker resource
 2. Define topics
-3. Connect services to the message broker and relevant topics
-4. Implement event production and consumption logic
+3. Install a message broker plugin
```

Comment:
@dericksozo - As one needs to install a plugin for each Service, I wonder if 3 and 4 should switch- so one needs first to connect services to the message broker and relevant topics, and then install the plugin in the relevant services. WDYT?